editor/utils/shortcuts.ts

```diff
@@ -47,3 +52,21 @@ export const useKeyboardShortcut = (
     }
   }, [handleKeyPress]);
 };
+
+// Handles scrolling down to element on key press
+export const scrollToElementOnKeyPress = (
+  event: KeyboardEvent,
+  shortcutKeyLookup?: ShortcutKeyLookup[]
+) => {
+  const eventKey = event.key.toUpperCase();
+  const elementId =
+    shortcutKeyLookup &&
+    shortcutKeyLookup.find((shortcut) => eventKey === shortcut.key)?.elementId;
+  const element = document.getElementById(String(elementId));
+  element?.scrollIntoView({ behavior: "instant" });
+};
+
+// Handles scrolling to top of page
+export const scrollToTop = () => {
```

Comment:
honestly instead of making another `scrollToTop` function , i could probably use `scrollToElementOnKeyPress` and pass `id="top"` to the main content `div` element in `SidebarLayout` and add "top" to the `shortcutKey` lookup object in there, but i wanted to show how we can use diff functions to handle key presses differently, if in the future we want to do something else w them thats not just scrolling in a page  :shrug: 

so i can remove this and use `scrollToElementOnKeyPress` but wanted to provide an example